### PR TITLE
Docker hardening

### PR DESCRIPTION
**Pull request by breathbath**

### CODE REVIEW

**Review by Andrea-Campanella — APPROVED** [review]
LGTM @kuujo @pierventre @hwchiu can you please take a look too ?

**Comment by breathbath** [comment]
> [Coverage Status](https://coveralls.io/builds/34915859)
> 
> Coverage decreased (-0.003%) to 54.174% when pulling **[7168198](https://github.com/atomix/atomix/commit/7168198e1330d7e1f9066f3c29c376ffc862d8c3) on breathbath:master** into **[842276c](https://github.com/atomix/atomix/commit/842276c02541267a61bca47f82c2e9d740245fcb) on atomix:master**.

Code coverage analytics should not include Dockerfiles

File: dist/Dockerfile [hunk 1]
```diff
@@ -4,12 +4,23 @@ FROM openjdk:8
...
+ENV ATOMIX_DIR=/var/lib/atomix
+ENV DATA_DIR=${ATOMIX_DIR}/data
+
 ARG VERSION
-RUN mkdir -p /opt/atomix
-COPY target/atomix.tar.gz /opt/atomix/atomix.tar.gz
-RUN tar -xvf /opt/atomix/atomix.tar.gz -C /opt/atomix && rm /opt/atomix/atomix.tar.gz
 
-WORKDIR /opt/atomix
+RUN groupadd -g 2000 atomix && \
```
Comment (johnou) [inline dist/Dockerfile:13]: might be safer avoid hard coded group and user ids.

Comment (breathbath) [reply dist/Dockerfile:13]: if we don't hard code it where do we have this value from?

Comment (johnou) [reply dist/Dockerfile:13]: Use names instead?

**Pull request closed without merging**